movie.sh

```diff
@@ -1,15 +1,16 @@
 #!bin/bash
 
-mkdir tmp #Making a folder to dump files
+readonly TEMP_PATH="/var/folders/3c/gjd471t51px36pjlbr7zggmh0000gn/T/imdb"
+mkdir $TEMP_PATH      # Making a Folder to Dump Files
```

Comment:
I got the error - Basically I've written my tmp folder path. I will send the new code with the fix using the Environment Variable.
Thanks